tools/fuchsia/gen_package.py

```diff
@@ -93,8 +93,6 @@ def main():
       output_dir,
       '-n',
       args.far_name,
-      '-k',
-      args.signing_key,
```

Comment:
I think `args.signing_key` is unused elsewhere in this file so you can delete it from the program's arguments and here: https://cs.opensource.google/flutter/engine/+/master:tools/fuchsia/fuchsia_archive.gni;l=116-117.

Reply:
done.